DataConnectors/Digital shadows/DigitalShadowsConnectorAzureFunction/DS_poller.py

```diff
@@ -41,10 +48,21 @@ def get_data(self):
         """
         triage_id = []
         try:
-            event_dataJSON = self.DS_obj.get_triage_events(str(self.before_time), str(self.after_time))
-            event_data = json.loads(event_dataJSON)
+            if(isinstance(self.event, int)):
+                event_dataJSON = self.DS_obj.get_triage_events_by_num(self.event)
+                event_data = json.loads(event_dataJSON)
+                self.date.post_event(self.event + int(len(event_data[:20])))
+            else:
+                event_dataJSON = self.DS_obj.get_triage_events(str(self.before_time), str(self.after_time))
```

Comment:
Do we need to convert the time to string? is it not already a string. If not, what format should we convert it to?

Reply:
yes it has to be a string. I can remove the str type casting since it is already a string.